docker-compose-prod.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3.9'
```

Comment:
to be tested
As the only difference with the dev docker file is a different config file, I don't expect particular issues